bitcoin/src/consensus/params.rs

```diff
@@ -88,8 +88,8 @@ impl Params {
                 bip66_height: 1,
                 rule_change_activation_threshold: 1916, // 95%
                 miner_confirmation_window: 2016,
-                pow_limit: Work::SIGNET_MIN,
-                pow_target_spacing: 10 * 60,            // 10 minutes.
+                pow_limit: Target::SIGNET_MAX,
+                npow_target_spacing: 10 * 60,            // 10 minutes.
```

Comment:
```suggestion
                pow_target_spacing: 10 * 60,            // 10 minutes.
```

Reply:
Reminds me of one of the more painful review comments I once got early in my days as an open source hacker, "does this even compile?"